src/RealtimeEffectPanel.cpp

```diff
@@ -290,8 +288,8 @@ namespace
       std::shared_ptr<RealtimeEffectState> mEffectState;
       std::shared_ptr<EffectSettingsAccess> mSettingsAccess;
 
-      ThemedButtonWrapper<wxButton>* mChangeButton{nullptr};
-      ThemedButtonWrapper<wxBitmapButton>* mEnableButton{nullptr};
+      ThemedAButtonWrapper<AButton>* mChangeButton{nullptr};
+      AButton* mEnableButton{nullptr};
```

Comment:
The power buttons are just `AButton` and not themable.  Does this matter?  Are the images really the same in all themes?

Reply:
It is themable, but stored as a `AButton*`